horace_core/Tobyfit/instpars_DGdisk.m

```diff
@@ -42,7 +42,7 @@
 
     inst=repmat({instruments{1}},[nrun,1]);
     for i=2:nrun
-        inst{i}=instruments{i};;
+        inst{i}=instruments{i};
     end
```

Comment:
What is the difference between this and `inst = instruments(1:nrun)`

Reply:
this is the original code. Your comment sees correct; it is possible this is teed up to add extra manipulation of the instruments. The code will be replaced when the new code is bedded down and connected up to nxspe.

The second ; has been removed